vignettes/immunedeconv2_vignette.Rmd

```diff
@@ -135,26 +139,44 @@ knitr::kable()
 Here are some further ideas of how you could plot your deconvolution results. 
 
 ```{r plot, fig.width=7, fig.height=4}
-df <- immunedeconv2::deconvolute(bulk_small, signatureMatrix, "bisque", single_cell_data_small, cell_type_annotations_small) %>%
-reshape2::melt()
-ggplot(df, aes(y=Var2, x=value, fill=Var1))+geom_bar(stat="identity", position = "stack")+labs(title="Bisque", y="sample", x="proportion", fill="cell type")+scale_fill_brewer(palette="Accent", breaks=c("B cell","Monocyte conventional","NK cell","T cell CD4","T cell CD8"))
+res_bisque <- immunedeconv2::deconvolute(bulk, signatureMatrix, "bisque", single_cell_data, cell_type_annotations)
+reshape2::melt(res_bisque) %>% set_colnames(c("samples", "cell_type", "predicted_fraction")) %>%
```

Comment:
In the future, consider using `pivot_longer` and `pivot_wider` from the tidyr package. `reshape2` is no longer maintained.